packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -318,46 +320,33 @@ export abstract class Chart extends Observable implements AgChartInstance {
-                return;
-            }
-
-            this._lastAutoSize = [width, height];
-            this.resize(undefined, undefined, 'SizeMonitor');
-        });
-        // eslint-disable-next-line sonarjs/no-duplicate-string
-        this.layoutService.addListener('start-layout', (e) => this.positionPadding(e.shrinkRect));
-        this.layoutService.addListener('start-layout', (e) => this.positionCaptions(e.shrinkRect));
-        this.layoutService.addListener('layout-complete', (e) => this.layoutComplete(e));
-
-        // Add interaction listeners last so child components are registered first.
-        this.interactionManager.addListener('click', (event) => this.onClick(event));
-        this.interactionManager.addListener('dblclick', (event) => this.onDoubleClick(event));
-        this.interactionManager.addListener('hover', (event) => this.onMouseMove(event));
-        this.interactionManager.addListener('leave', (event) => this.onLeave(event));
-        this.interactionManager.addListener('page-left', () => this.destroy());
-        this.interactionManager.addListener('wheel', () => this.disablePointer());
-
-        this.animationManager.addListener('animation-frame', (_) => {
-            this.update(ChartUpdateType.SCENE_RENDER);
-        });
-        this.highlightManager.addListener('highlight-change', (event) => this.changeHighlightDatum(event));
-        this.zoomManager.addListener('zoom-change', (_) =>
-            this.update(ChartUpdateType.PROCESS_DATA, { forceNodeDataRefresh: true })
+        SizeMonitor.observe(this.element, (size) => this.rawResize(size));
+        this._destroyFns.push(
+            // eslint-disable-next-line sonarjs/no-duplicate-string
+            this.layoutService.addListener('start-layout', (e) => this.positionPadding(e.shrinkRect)),
+            this.layoutService.addListener('start-layout', (e) => this.positionCaptions(e.shrinkRect)),
+            this.layoutService.addListener('layout-complete', (e) => this.layoutComplete(e)),
+
+            // Add interaction listeners last so child components are registered first.
+            this.interactionManager.addListener('click', (event) => this.onClick(event)),
+            this.interactionManager.addListener('dblclick', (event) => this.onDoubleClick(event)),
+            this.interactionManager.addListener('hover', (event) => this.onMouseMove(event)),
+            this.interactionManager.addListener('leave', (event) => this.onLeave(event)),
+            this.interactionManager.addListener('page-left', () => this.destroy()),
+            this.interactionManager.addListener('wheel', () => this.disablePointer()),
+            this.interactionManager.addListener('wheel', () => this.disablePointer()),
+
+            // Block redundant and interfering attempts to update the hovered element during dragging.
+            this.interactionManager.addListener('drag-start', () => (this.reapplyPointerOnPerformUpdate = false)),
```

Comment:
This is similar to the animation's `disableInteraction` feature. Definitely different in behaviour from that specific case atm, but maybe if we do more similar things in the future we may want to look into a way of generically disabling certain or all interactions?